core/domain/exp_domain.py

```diff
@@ -2404,10 +2409,12 @@ def from_yaml(cls, exploration_id, yaml_content):
             exploration_dict['auto_tts_enabled'],
             exploration_dict['correctness_feedback_enabled'])
 
+        # Construcotr calculates the proto_size_in_bytes and add it to the
```

Comment:
Constructotr --> The constructor

add --> adds

Last sentence --> So, we add that value directly to the dict.